.release/security-scan.hcl

```diff
@@ -2,9 +2,20 @@
 # SPDX-License-Identifier: MPL-2.0
 
 binary {
-	secrets      = true
-	go_modules   = true
-	osv          = true
-	oss_index    = false
-	nvd          = false
+  secrets      = true
+  go_modules   = true
```

Comment:
We might want to also add `go_stdlib = true` to catch issues like [GO-2025-3420](https://pkg.go.dev/vuln/GO-2025-3420).